src/node_file.cc

```diff
@@ -414,18 +416,22 @@ inline int SyncCall(Environment* env, Local<Value> ctx, fs_req_wrap* req_wrap,
 void Access(const FunctionCallbackInfo<Value>& args) {
   Environment* env = Environment::GetCurrent(args.GetIsolate());
   HandleScope scope(env->isolate());
-  Local<Context> context = env->context();
-  CHECK_GE(args.Length(), 2);
+
+  int argc = args.Length();
```

Comment:
`const int argc = ...` if you're caching it.  `.Length()` is really just an inexpensive pointer load though.